assets/styles/_editorMenuBar.scss

```diff
@@ -0,0 +1,39 @@
+$color-white: #fff;
+$color-black: #000;
+
+.menubar {
+  display: flex;
+  margin-bottom: 1rem;
+  transition: visibility 0.2s 0.4s, opacity 0.2s 0.4s;
+
+  &.is-hidden {
+    visibility: hidden;
+    opacity: 0;
+  }
+
+  &.is-focused {
+    visibility: visible;
+    opacity: 1;
+    transition: visibility 0.2s, opacity 0.2s;
+  }
+
+  &__button {
+    font-weight: bold;
+    display: inline-flex;
+    background: transparent;
+    border: 0;
+    color: $color-black;
+    padding: 0.2rem 0.5rem;
+    margin-right: 0.2rem;
+    border-radius: 3px;
```

Comment:
Ich würde für `border-radius` in der `_variables.scss` mal eine Variable `$border-radius` anlegen, brauchen wir ja an mehreren Stellen.

Reply:
done